lib/rucio/core/credential.py

```diff
@@ -59,9 +69,12 @@ def get_signed_url(service, operation, url, lifetime=600):
         raise UnsupportedOperation('URL must not be empty')
 
     if lifetime:
-        if not isinstance(lifetime, (int, long)):
+        if not isinstance(lifetime, integer_types):
             try:
-                lifetime = long(lifetime)
+                if PY2:
+                    lifetime = long(lifetime)
```

Comment:
I guess the `long` type is needed here? Therefor I created an exception in the python3 check to exclude the long check, because otherwise the test will fail. But then it is possible that someone will use `long` somewhere else without the python version check.
There is the possbility to use `form builtins import int` to have a solution for py2 and py3 but its causing problems with python2.6 (see #1845 )

Reply:
Python 2.6 is not an issue here, since this is only used in the server. I think it is fine like that, but @mlassnig should comment.

Reply:
I don't think that setting lifetimes longer than a couple million years will be problematic ;-)